spec/integration/api_resources_controller_spec.rb

```diff
@@ -38,4 +38,63 @@
       end
     end
   end
+
+  path '/api/learning_units/{learning_unit_id}/resources' do
+    get 'List of all resources belonging to a learning unit' do
+      tags 'Resources'
+      operationId 'listResources'
+      produces 'application/json'
+      parameter name: :learning_unit_id, in: :path, type: :string
+
+      response '200', 'Success' do
+        schema type: :array, items: {
+          properties: {
+            id: { type: :integer },
+            name: { type: :string },
+            url: { type: :string },
+            average_evaluation: { type: :string }
+          }
+        }
+        let(:learning_unit) { create(:learning_unit) }
+        let(:learning_unit_id) { learning_unit.id }
```

Comment:
Not that important, but considering that you are only using the learning_unit_id variable, you could reduce this to a one-liner `let(:learning_unit_id) { create(:learning_unit).id }`

Reply:
I use the `:learning_unit` to create a new resource (line 77)

Reply:
Done!